src/models/ingredient.js

```diff
@@ -0,0 +1,23 @@
+const { Schema, model, SchemaTypes } = require("mongoose");
+
+const schema = new Schema({
+    
+    IngredientId: {
```

Comment:
Please stick to convention: do not use capital letters at the start of schema properties and/or do them in class.

Reply:
Also, do not append 'id' to every stored reference. Later on, you'll want to populate this field with data from another collection, then you find yourself stuck with an object named with a very weird xxId name. There are others, but these two reasons seem like they're the most important.